src/services/dashboards/dashboard-detail/lib/type.ts

```diff
@@ -1,8 +1,29 @@
+import type { TimeStamp } from '@/models';
+
+import type { DashboardScope } from '@/services/dashboards/type';
 import type { WidgetConfig } from '@/services/dashboards/widgets/config';
 import type { WidgetTheme } from '@/services/dashboards/widgets/view-config';
 
-
+// WIDTH
 export type WidgetThemeOption = WidgetConfig['theme'];
-
 // ['violet' | 'blue' | 'coral' ... | undefined]
 export type WidgetThemeAssignedList = Array<WidgetTheme | undefined>;
+
+
+// Model
+export interface DashboardDetailDataModel {
+    'domain_dashboard_id': string
+    'name': string
+    'viewers': DashboardScope
+    'version': number
+    'layouts': Array<any>
+    'dashboard_options': any
+    'settings': any
+    'dashboard_options_schema': any
+    'labels': Array<string>
+    'tags': any
+    'user_id': string
+    'domain_id': string
+    'created_at': TimeStamp
+    'updated_at': TimeStamp
+}
```

Comment:
suggestion: DashboardModel is already exist. Also, in [this PR](https://github.com/cloudforet-io/console/pull/491), I updated the DashboardModel. Please rebase it after I merge the PR above! 🙏🙏

Reply:
Oh I got it !